src/fsharp/FSharp.Build/Microsoft.FSharp.targets

```diff
@@ -221,7 +221,7 @@ this file.
               Prefer32Bit="$(Actual32Bit)"
               References="@(ReferencePath)"
               ReferencePath="$(ReferencePath)"
-              Resources="@(_CoreCompileResourceInputs);@(CompiledLicenseFile);@(AdditionalEmbeddedResource)"
+              Resources="@(_CoreCompileResourceInputs);@(ManifestResourceWithNoCulture);@(ManifestNonResxWithNoCultureOnDisk);@(CompiledLicenseFile);@(AdditionalEmbeddedResource)"
```

Comment:
@jack-pappas This fix doesn't work on  .NET Framework on Windows, perhaps we should make it conditional on Mono